src/main/java/org/folio/circulation/domain/policy/library/ClosedLibraryStrategyService.java

```diff
@@ -35,6 +36,16 @@ public ClosedLibraryStrategyService(
     this.isRenewal = isRenewal;
   }
 
+  public CompletableFuture<Result<LoanAndRelatedRecords>> applyCLDDM(
```

Comment:
I prefer the use of words rather than an abbreviation. Was this done to help this fit on a single line when called?

Reply:
Yes. AFAIR this method had more parameters in the past and didn't fit a single line. Now I tried to rename it to `applyClosedLibraryDueDateManagement` and seems like it fits.